tests/proofs/liquidity-baking/lb-compiled.md

```diff
@@ -26,7 +26,6 @@ Since we work with specific code that contains a finite number of annotations, w
 
 ```k
   syntax FieldAnnotation ::= "%deadline"
-                           | "%default"
                            | "%minTokensBought"
                            | "%mintOrBurn"
                            | "%quantity"
```

Comment:
Use `[token]` and drop the macros:

```suggestion
                           | "%minTokensBought" [token]
                           | "%mintOrBurn"           [token]
                           | "%quantity"                 [token]
```

Don't know why I can suggest for the whole block.

Reply:
I replaced the macros with token annotations as you asked.